### ISSUE TITLE
Command and Method to crete Upgrade Schema

### ISSUE THREAD

**Issue by astorm** [astorm/pestle#209]

**Comment by astorm** [comment]
Dev version only until the official release, but usage is 

    pestle_dev magento2:generate:schema-upgrade Pulsestorm_Blog 1.1.1

This will 

- create a UpgradeSchema and UpgradeData class in the specified module
- these classes will use a Package\Vendor\Setup\Scripts helper, also generated
- increment module.xml version
- create a data and schema x.x.x.php scripts in top-level-of-module-folder `upgrade_scripts`

The mentioned helper implements a Magento 1 like setup-resource/migration script routine.  

In other words, run this command, add your SQL upgrading code to the generated script file, and then run `bin/magento setup:upgrade`.  No more sketchy `version_compare` for you.
